bouffalo-hal/src/sec/aes.rs

```diff
@@ -471,20 +466,25 @@ pub enum XtsMode {
 pub struct SecureBoot(u32);
 
 impl SecureBoot {
-    // TODO
-    // const SECURE_BOOT_KEY_SELECT: u32 = 1 << 0;
+    const SECURE_BOOT_KEY_SELECT: u32 = 1 << 0;
     const XTS_MODE: u32 = 1 << 15;
     const XTS_DATA_UINT_LENGTH: u32 = 0xffff << 16;
 
-    // TODO
-    // /// Set secure boot key selection
-    // #[inline]
-    // pub fn set_secure_boot_key_select(&mut self, sboot_key_select: SecureBootKeySelect) {}
+    /// Set secure boot key selection
+    #[inline]
+    pub fn set_secure_boot_key_select(&mut self, sboot_key_select: bool) {
+        if sboot_key_select {
+            self.0 |= Self::SECURE_BOOT_KEY_SELECT;
+        } else {
+            self.0 &= !Self::SECURE_BOOT_KEY_SELECT;
+        }
+    }
 
-    // TODO
-    // /// Get current secure boot key selection
-    // #[inline]
-    // pub fn get_secure_boot_key_select(&self) -> SecureBootKeySelect {}
+    /// Get current secure boot key selection
+    #[inline]
+    pub fn get_secure_boot_key_select(&self) -> bool {
+        (self.0 & Self::SECURE_BOOT_KEY_SELECT) != 0
+    }
```

Comment:
```suggestion
    /// Get current secure boot key selection.
    #[inline]
    pub fn secure_boot_key_select(&self) -> bool {
        (self.0 & Self::SECURE_BOOT_KEY_SELECT) != 0
    }
```

Rust in-line documents would form complete English sentences including the period `.` on the tail.